lib/src/api/engine/any/native.rs

```diff
@@ -174,14 +174,14 @@ impl Connection for Any {
 						let maybe_connector = address.config.tls_config.map(Connector::from);
 						#[cfg(not(any(feature = "native-tls", feature = "rustls")))]
 						let maybe_connector = None;
+						#[allow(deprecated)]
 						let config = WebSocketConfig {
-							max_send_queue: match capacity {
-								0 => None,
-								capacity => Some(capacity),
-							},
+							max_send_queue: None,
```

Comment:
Why did this change to None? Could you revert it please?

Reply:
Per the tungstenite 0.20.1 documentation, the max_send_queue field is deprecated and no longer performs any function.

[source](https://docs.rs/tungstenite/latest/tungstenite/protocol/struct.WebSocketConfig.html#structfield.max_send_queue)

Reply:
In that case could we either describe the property as deprecated (if it needs to be included in the API due to backwards compatiility), or remove if it doesnt need to be backwards compatible please 🙏

Reply:
I'm not completely sure what the best approach would be in this case. What do you think?

Reply:
Re-reviewed, I thought WebSocketConfig was surrealdb, but its actually tungstenite.
In that case it can stay as-is. Thanks for this!